### PR TITLE
Finished feature/footer component

### PR DESCRIPTION
**Pull request by M4TThys123**

### CODE REVIEW

File: src/lib/components/footer.svelte [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+	@media (min-width: 60rem) {
+		footer {
+			padding: 1rem 2rem;
+		}
+	}
+
+	@media (min-width: 81.25rem) {
+		footer {
+			padding: 1rem 5rem;
```
Comment (boudewijnbout) [inline src/lib/components/footer.svelte:24]: Did you check that the padding is the same on for example the header?

**Review by boudewijnbout — CHANGES_REQUESTED** [review]

File: src/lib/components/footer.svelte [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<footer>
+	<span>© 2022 CMD AMSTERDAM</span>
```
Comment (DaanKorver) [inline src/lib/components/footer.svelte:2]: En ```<slot/>``` gebruiker aub

**Review by DaanKorver — CHANGES_REQUESTED** [review]

File: src/lib/components/footer.svelte (hunk 1 above)
Comment (M4TThys123) [reply src/lib/components/footer.svelte:24]: I'm changing it to:
```css
@media (max-width: 45rem) { 
}
```

**Review by boudewijnbout — APPROVED** [review]
LGTM!

**Review by DaanKorver — APPROVED** [review]

**Pull request merged**